redfish-core/lib/pcie_slots.hpp

```diff
@@ -280,6 +281,57 @@ inline void getLocationCode(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
         });
 }
 
+inline void
+    addLinkedFabricAdapter(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
+                           const std::string& pcieSlotPath, size_t index)
+{
+    constexpr std::array<std::string_view, 1> fabricAdapterInterfaces{
+        "xyz.openbmc_project.Inventory.Item.FabricAdapter"};
+    dbus::utility::getAssociatedSubTreePaths(
```

Comment:
Are we planning to upstream this code? The lambda function is too long and does not adhere to the coding guidelines.

Reply:
As this time, this is downstream only, as this association is a temporary workaround for 1050 or earlier releases.
That's why it is under OEM schema.